.github/workflows/verify.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
     name: Verify Deps and Codegen
     strategy:
       matrix:
-        go-version: [1.18.x]
+        go-version: [1.21.x]
```

Comment:
This looks like maybe something you accidentally included in the PR? If you remove the change to this file, I can merge the PR with the changes to the elections files.

Reply:
Though, not relevant to the PR without it the GH-actions are broken and the PR will not pass CI checks (https://github.com/knative/community/actions/runs/6542752164). I could do that PR separately, but as we'd like to proceed with the announcement I think we also can keep it in one. WDYT?

Reply:
Aha, I see the reason now. However, we'll need someone from the @knative/technical-oversight-committee to merge this one, since they are the OWNERS for the .github dir. Maybe @dsimansk, since it looks like maybe David is in a European time zone :) 

LGTM from an election standpoint.

Reply:
Sure, done.